skore/src/skore/sklearn/_cross_validation/report.py

```
@@ -163,9 +169,42 @@ def _fit_estimator_reports(self):
         )
 
         estimator_reports = []
-        for report in generator:
-            estimator_reports.append(report)
-            progress.update(task, advance=1, refresh=True)
+        try:
+            for report in generator:
+                estimator_reports.append(report)
+                progress.update(task, advance=1, refresh=True)
+        except KeyboardInterrupt:
+            from skore import console  # avoid circular import
+
+            console.print(
+                Panel(
+                    title="Cross-validation interrupted",
+                    renderable=(
+                        "Cross-validation process was interrupted manually before all "
+                        "estimators could be fitted; CrossValidationReport object "
+                        "might not contain all the expected results."
+                    ),
+                    style="orange1",
+                    border_style="cyan",
+                )
+            )
+
```

Can you add a comment in the class description that the `KeyboardInterrupt` exception is swallowed and will only interrupt the cross-validation, not the entire program.

---

I don't understand, do you mean you want https://github.com/probabl-ai/skore/pull/1287/files/1728f0ef202849e9e69389c15479a0ba51d9467c#diff-a54c7ccc0fb91b7c2e48130faa2a561ff6b783d44ab555339339a2a82ed7bf0dR33-R35 to be even clearer? or do you want another message entirely?

---

done!